docker/history/Readme.md

```diff
@@ -0,0 +1,64 @@
+
+# 🌟 Steem Account Data Updater
+
+This repository contains a script to update and maintain Steem account data and related properties in MongoDB. It fetches data from the Steem blockchain and stores it in MongoDB, running scheduled updates to keep the data current.
+
+## ✨ Features
+- Fetches Steem account details and global properties.
+- Stores data in MongoDB with structured logging and error handling.
+- Supports batch requests to the Steem API for efficient data retrieval.
+- Uses APScheduler for scheduled updates.
+
+## 📋 Requirements
+- Python 3.9
+- MongoDB
+- Docker (for containerized deployment)
+
+## ⚙️ Configuration
+Create a `config.json` file in the root directory with the following structure:
+
+```json
+{
+    "STEEMD_URLS": ["https://api.steemit.com"],
+    "MONGODB": "your_mongodb_connection_string"
+}
+```
+
+## 🚀 Installation
+
+1. Clone the repository:
+   ```sh
+   git clone https://github.com/your-repo/steem-account-updater.git
+   cd steem-account-updater
+   ```
+
+2. Build the Docker image:
+   ```sh
+   docker build -t steem-history .
+   ```
+
+3. Run the Docker container:
+   ```sh
+   docker run -d --name steem-history steem-history
```

Comment:
It need add `config.json` file in params.
like this
````
docker run -d --name steem-history -v ./config.json:/src/config.json steem-history
```

 But I suggest to also add environment configuration way.

Reply:
Volume flag is not required as config.json is also  copied inside the container , Script now can take steemd url and mongo as environment variable too ,  

```sh
   docker run -d --name steem-history -e STEEMD_URLS="http://10.10.100.12:8080" -e MONGODB="mongodb://10.10.100.30:27017" steem-history
   ```